src/main/markdown/doc/latest/tutorial/appengine.md

```diff
@@ -83,41 +83,42 @@ You will reference this configuration later by its name "transactions-optional".
 
 ```
 <!-- Configure path to GWT SDK -->
-    <property name="gwt.sdk" location="_Path to GWT_" />
-    <!-- Configure path to App Engine SDK -->
-    <property name="appengine.sdk" location="_Path to App Engine SDK_" />
+<property name="gwt.sdk" location="_Path to GWT_" />
+<!-- Configure path to App Engine SDK -->
+<property name="appengine.sdk" location="_Path to App Engine SDK_" />
 ```
 
 7.  Add a property for a App Engine tools class path.
 
-        <path id="project.class.path">
-        <pathelement location="war/WEB-INF/classes"/>
-        <pathelement location="${gwt.sdk}/gwt-user.jar"/>
-        <fileset dir="${gwt.sdk}" includes="gwt-dev*.jar"/>
-        <!-- Add any additional non-server libs (such as JUnit) -->
-        <fileset dir="war/WEB-INF/lib" includes="**/*.jar"/>
-        </path>
-
-        <path id="tools.class.path">
-        <path refid="project.class.path"/>
-        <pathelement location="${appengine.sdk}/lib/appengine-tools-api.jar"/>
-        <fileset dir="${appengine.sdk}/lib/tools">
-          <include name="**/asm-*.jar"/>
-          <include name="**/datanucleus-enhancer-*.jar"/>
-        </fileset>
-        </path>
+```
```

Comment:
remove the triple backtick here and keep the indentation, otherwise it will break the ordered list